plugins/openweather/game/config/openweather.yml

```diff
@@ -0,0 +1,88 @@
+---
+openweather:
+  api_key: YOUR_API_KEY
```

Comment:
Since some folks like to keep their configuration files in source control, you probably want to tell them to add their API key as a new config key in secrets.yml rather than here.  Then use `Global.read_config('secrets', 'openweather_api_key')` in your code.

Reply:
Much cleaner putting it in secrets.yml. Fixed.